docs/Manual/03 Schedules.md

```diff
@@ -16,39 +16,40 @@ for i in range(16):
 
 ### Iteration space slices
-*Iteration space slices* are an abstract concept that affects many different aspects of the Accera programming model. Since the iteration space dimensions are ordered, each element of the iteration space can be identified by a vector of coordinates. For example, the vector (5, 6, 7) identifies the iteration at position 5 along the first dimension, 6 along the second dimension, and 7 along the third dimension. If one or more of the coordinates is replaced with the *wildcard* symbol \*, we get an *iteration space slice*, which is a set of iterations obtained by replacing the wildcard with all possible values. For example, (\*, \*, 5) represents a slice that contains all the elements whose last coordinate is 5. The *dimension of a slice* equals the number of wildcards in its definition.
+*Iteration space slices* is an abstract concept that affects different aspects of the Accera programming model. Since the iteration space dimensions are ordered, each element of the iteration space can be identified by a vector of coordinates.  For example, the vector (5, 6, 7) identifies the iteration at position 5 along the first dimension, position 6 along the second dimension, and position 7 along the third dimension. If one or more coordinates are replaced with the *wildcard* symbol `\*`, we get an *iteration space slice*, which is a set of iterations obtained by replacing the wildcard with all possible values. For example, (\*, \*, 5) represents a slice containing all the elements with 5 as their last coordinate. The *dimension of a slice* equals the number of wildcards in its definition.
 
 ### Loops, indices, and dimensions
-When we defined `nest`, we used variables such as `i`, `j`, `k` to name the loops in the loop-nest. When we described the default schedule using equivalent for-loops, `i`, `j`, and `k` became the index variables of those loops. When we represent a schedule as an iteration space, these variables are used as the names of the corresponding iteration space dimensions. From here on, we move seamlessly between these different representations and use the terms *loop*, *index*, and *dimension* interchangeably.
+When we defined `nest`, we used variables such as `i`, `j`, and `k` to name the loops in the loop-nest. When we described the default schedule using equivalent for-loops, `i`, `j`, and `k` became the index variables of those loops. When we represented a schedule as an iteration space, these variables were used as the names of the corresponding iteration space dimensions. From here on, we move seamlessly between these different representations and use the terms *loop*, *index*, and *dimension* interchangeably.
```

Comment:
"represent" and "are" are actually ok here because the topic of this section is about schedules, and how a schedule can represent an iteration spaces and change how we interpret the variables `i`, `j`, and `k`. You bring up a good point though. Maybe we need to add a "Now" to make that clear, e.g. 

"Now, when we represent a schedule as an iteration space, these variables are used as ..."